services/web-app/components/mdx-icon/mdx-icon.js

```diff
@@ -42,7 +42,8 @@ const svgIcons = {
   zenhub: Zenhub,
   vue: Vue,
   svelte: Svelte,
-  js: Js
+  js: Js,
+  vanilla: Js
```

Comment:
Do we have any react-native libraries? It would just be a React logo right?

Reply:
I don't think we do at the moment, but it's in our schema so I think we should handle it just in case.
Yeah, I think it's the same react logo so maybe we should just map it to react?

Reply:
Updated @francinelucca